Lib/test/test_configparser.py

```diff
@@ -980,6 +980,39 @@ def test_set_nonstring_types(self):
         self.assertRaises(TypeError, cf.set, "sect", 123, "invalid opt name!")
         self.assertRaises(TypeError, cf.add_section, 123)
 
+    def test_str_and_repr(self):
```

Comment:
Let's have separate tests (one for `str`, one for `repr`)

Reply:
Done